doc/Learn/Mvux/Tutorials/HowTo-ListFeed.md

```diff
@@ -2,261 +2,150 @@
 uid: Uno.Extensions.Mvux.HowToListFeed
 ---
 
-# How to create a list feed
+# MVUX List Feed
 
-In this tutorial, you will learn how to create an MVUX project that asynchronously requests
-and displays a collection of items from a service, and enables refreshing the data.
+Quick guide for exposing a collection through `IListFeed<T>` and rendering it with `FeedView`.
 
-In this tutorial, you will learn how to create an MVUX project and the basic usage of a list-feed (`IListFeed<T>`), and the `FeedView` control.
+## TL;DR
```

Comment:
We need to make scenarios that lead by the outcome. Creating a list is not an outcome, it's the result. Let's name it some other way.